python/semantic_kernel/schema/kernel_json_schema_builder.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import Any, Type, get_type_hints
+
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
+
+class KernelJsonSchemaBuilder:
+
+    TYPE_MAPPING = {
+        int: "integer",
+        str: "string",
+        bool: "boolean",
+        float: "number",
+        list: "array",
+        dict: "object",
+        "int": "integer",
+        "str": "string",
+        "bool": "boolean",
+        "float": "number",
+        "list": "array",
+        "dict": "object",
+        "object": "object",
+    }
+
+    @classmethod
+    def build(cls, parameter_type: Type, description: str | None = None) -> dict[str, Any]:
+        """Builds JSON schema for a given parameter type."""
+        if issubclass(parameter_type, KernelBaseModel):
+            return KernelJsonSchemaBuilder.build_model_schema(parameter_type, description)
+        elif hasattr(parameter_type, "__annotations__"):
```

Comment:
You can maybe combine these two? Or at least just use if instead of elif since you return